spec/features/user_registration_spec.rb

```diff
@@ -69,40 +70,40 @@ def signup(username, email, password)
  
 describe "Signing in" do
 
-describe "successfully" do
-  let!(:user) { FactoryGirl.create(:user) } 
-  it "displays a flash message indicating success" do
-    visit user_session_path #visit users/sign_in page
-    fill_in 'Email',    with: user.email
-    fill_in 'Password', with: user.password
-    click_button "Sign In"
-    expect(current_path).to eq "/"
-    expect(page).to have_content("Hello")
+  describe "successfully" do
+    let!(:user) { FactoryGirl.create(:user) } 
+    it "displays a flash message indicating success" do
+      visit user_session_path #visit users/sign_in page
```

Comment:
Ok I will look at this one and that might be a good idea based in where I am in project in general

I will make appropriate changes and merge

Sent from my iPhone

> On Mar 18, 2015, at 10:45 AM, Levi Cole <notifications@github.com> wrote:
> 
> In spec/features/user_registration_spec.rb:
> 
> > @@ -69,40 +70,40 @@ def signup(username, email, password)
> >   
> >  describe "Signing in" do
> >  
> > -describe "successfully" do
> > -  let!(:user) { FactoryGirl.create(:user) } 
> > -  it "displays a flash message indicating success" do
> > -    visit user_session_path #visit users/sign_in page
> > -    fill_in 'Email',    with: user.email
> > -    fill_in 'Password', with: user.password
> > -    click_button "Sign In"
> > -    expect(current_path).to eq "/"
> > -    expect(page).to have_content("Hello")
> > +  describe "successfully" do
> > +    let!(:user) { FactoryGirl.create(:user) } 
> > +    it "displays a flash message indicating success" do
> > +      visit user_session_path #visit users/sign_in page
> Also, we can just keep track of all the things we want to refactor and then have a refactoring session next week if you want! :)
> 
> —
> Reply to this email directly or view it on GitHub.
>